packages/playwright-test/src/plugins/vitePlugin.ts

```diff
@@ -37,42 +40,68 @@ export function createPlugin(
       const use = config.projects[0].use as any;
       const viteConfig: InlineConfig = use.viteConfig || {};
       const port = use.vitePort || 3100;
-
       configDir = configDirectory;
-
       process.env.PLAYWRIGHT_VITE_COMPONENTS_BASE_URL = `http://localhost:${port}/playwright/index.html`;
 
-      viteConfig.root = viteConfig.root || configDir;
-      viteConfig.plugins = viteConfig.plugins || [
-        frameworkPluginFactory()
-      ];
-      const files = new Set<string>();
-      for (const project of suite.suites) {
-        for (const file of project.suites)
-          files.add(file.location!.file);
+      const rootDir = viteConfig.root || configDir;
+      const outDir = viteConfig?.build?.outDir || path.join(rootDir, 'playwright', '.cache');
```

Comment:
maybe `.playwright/cache` is better than `playwright/.cache`.

Reply:
It is, but there is /playwright next to it anyways...